source/setting_descriptions.cpp

```diff
@@ -1420,40 +1416,262 @@ string_view GlitchHoverDescIntermediate               = "Some hovers may require
-string_view GlitchMegaflipDescNovice                  = "Only Megaflipping with bombs is required.";       //
-string_view GlitchMegaflipDescIntermediate            = "Some Megaflips where you don't have a lot of room\n"
-                                                        "to work with and positioning is more important\n" //
+/* Bomb OI */                                                                                              //
+string_view GlitchBombOIDescDisabled                  = "Allowing a bomb to explode in Link's hands while\n"
+                                                        "moving and then attempting to pull out a cutscene\n"
+                                                        "item on a specific frame will cause Link to play\n"
+                                                        "an invisible ocarina instead.\n"                  //
+                                                        "This will only work if bombs are not dimmed on the"
+                                                        "frame you use the cutscene item.";                //
+string_view GlitchBombOIDescNovice                    = "You may be expected to use ocarina items with a\n"//
+                                                        "bomb to play warp songs.";                        //
+string_view GlitchBombOIDescIntermediate              = "You may be expected to use ocarina items with a\n"//
+                                                        "bomb to play the ocarina where Link's position\n" //
+                                                        "matters.";                                        //
+string_view GlitchBombOIDescAdvanced                  = "You may be expected to use restricted items to use"
+                                                        "the cutscene item or to make the bombs usable on\n"
+                                                        "the correct frame.";                              //
+string_view GlitchBombOIDescExpert                    = "You may be expected to use restricted items to\n" //
+                                                        "perform ocarina items where Link's position isn't\n"
+                                                        "particularly lenient";                            //
+/* Hover Boost */                                                                                          //
+string_view GlitchHoverBoostDescDisabled              = "Equipping hover boots when Link takes damage will\n"
+                                                        "cause him to keep the high knockback speed and\n" //
+                                                        "lets him traverse large gaps.\n"                  //
+                                                        "If performed at the edge of a platform Link will\n"
+                                                        "instead perform a super jump which has less range\n"
```

Comment:
```suggestion
                                                        "instead perform a mega jump which has less range\n"
```
That's usually called Mega Jump